packages/common/src/core/types/IHubEntityManager.ts

```diff
@@ -47,8 +47,8 @@ export interface IHubEntityManager<T> {
    * @param filter
    * @param opts
    */
-  // search(
-  //   filter: Filter<"content">,
-  //   opts: IHubSearchOptions
-  // ): Promise<ISearchResponse<T>>;
+  search(
```

Comment:
do we need to mark this as optional? I guess when not yet implemented for a specific manager they could always throw?